### ISSUE TITLE
Rule proposal: Doc comment Asterix prefix style

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/eslint-plugin-unicorn#1607]
### Description

I prefer not using Asterix prefix on doc comments as I find them noisy and they also make it much harder to edit comments.

I'm aware you can almost get this with the JSDoc ESLint plugin, but it's a bit heavy to add that just for this and doc comments do not necessarily need to be JSDoc, so this rule would be more general.

### Fail

```js
/**
 * Add two numbers.
 * @param {number} num1 The first number.
 * @param {number} num2 The second number.
 * @returns {number} The sum of the two numbers.
 */
```

### Pass

```js
/**
Add two numbers.
@param {number} num1 The first number.
@param {number} num2 The second number.
@returns {number} The sum of the two numbers.
*/
```

(Notice how `*/` moved too)

**Comment by sindresorhus** [comment]
This is now accepted.

**Comment by papb** [comment]
[You probably meant asterisk](https://en.wikipedia.org/wiki/Asterix), right? 😆

**Comment by sindresorhus** [comment]
LOL, yes.
